users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -201,10 +207,17 @@ def get_returnn_config(
                 network.pop(layer)
         network["source"] = {"class": "copy", "from": "features"}
     else:
-        network["source"] = specaug_layer_jingjing(in_layer=["features"])
+        if specaug_mask_sorting:
+            network["features"]["subnetwork"]["specaug"] = specaug_layer_sorted(in_layer=["conv_h_act"])
```

Comment:
The specaug layer is added to the network here, but no other layer uses it as "from". So it will not be used. This needs to be fixed here.

Reply:
In addition (and this explains the memory issues that you had), the network dict still contains
```
"source": {"class": "copy", "from": "data"},
```
Before, this was overwritten by specaugment with input "features". Now, the whole feature subnetwork is completely unused. Therefore, the conformer operates on very long sequences causing issues for the memory.